ReleaseTooling/Sources/Utils/ShellUtils.swift

```diff
@@ -65,7 +65,10 @@ public extension Shell {
       let contents = """
       export PATH="/usr/local/bin:/usr/local/git/current/bin:$PATH"
       export LANG="en_US.UTF-8"
-      source ~/.bash_profile
+      BASH_PROFILE_PATH="~/.bash_profile"
+      if [ -f "$BASH_PROFILE_PATH" ]; then
+        source $BASH_PROFILE_PATH
+      fi
```

Comment:
Is there a simple command I could run to test this out? Specifically one that uses CocoaPods, since my `.bash_profile` is empty and `pod` is in `/Users/andrewheard/.gem/ruby/2.7.7/bin/pod` (not part of `PATH` specified above).

Reply:
> Hmm, if it's not needed, it would be more robust to remove.

I could switch to removing it and see if anything breaks on CI or locally when we run release commands. 

I think the original intent of this line was to source the user's `.bash_profile` so the PATH that the shell accesses would get access to any of the user's PATH modifications (e.g. adding Ruby to the path). I don't think this assumption is necessary anymore though.

Reply:
Oh, the ruby on the path may still be relevant.

I'm fine with the PR as originally written - with a comment to say that reason.

Reply:
Thanks, @andrewheard. Hmm, I'll merge this then and kick off a global CI run on a new PR that removes it completely. Pending the results there, we could clean things up by removing it.